book/getting-started/vagrant-setup.md

```diff
@@ -26,6 +24,28 @@ rm wallaroo-{{ book.wallaroo_version }}.tar.gz
 cd wallaroo-{{ book.wallaroo_version }}
 ```
 
+### Windows via Powershell
+
+**Note:** This section of the guide assumes you are using Powershell.
+
+Download and install git from [Git for Windows](https://gitforwindows.org/) and install it.
+
+If you haven't already done so, create a directory called `~/wallaroo-tutorial` and navigate there by running:
+
+```bash
+cd ~/
+mkdir ~/wallaroo-tutorial
+cd ~/wallaroo-tutorial
+```
+
+This will be our base directory in what follows. If you haven't already cloned the Wallaroo repo, do so now (this will create a subdirectory called `wallaroo-{{ book.wallaroo_version }}`):
+
+```bash
+git clone https://github.com/WallarooLabs/wallaroo wallaroo-{{ book.wallaroo_version }}
```

Comment:
In most places in the tutorial the name of the repo is just wallaroo and not `wallaroo-{{ book.wallaroo_version }}`. Should we keep that convention?

Reply:
I'm not sure what places you're referring to because the we switched from plain `wallaroo` to `wallaroo-{{ book.wallaroo_version }}` in release `0.5.2`.

Can you refer me to specific instances in the current version of the book?

Reply:
Oops you're right. nvm :smile: